components/driveTrain.py

```diff
@@ -66,12 +74,45 @@ def disableCreeperMode(self):
         self.driveMotorsMultiplier = self.prevMultiplier
         self.creeperMode = False
 
-    def stop(self, coast=False):
+    def stop(self):
         self.controlMode = ControlMode.kDisabled
 
     def getMeasuredSpeed(self):
         pass
 
+    def getRightSideDistTraveled(self):
+        """
+        Returns the right motor's distance traveled in inches
+        """
+        self.rightDistInch = (self.rightMotor.getPosition(0, positionUnits.kRotations) / self.gearRatio) * self.wheelCircumference
```

Comment:
Consider a function for this in the future where the math can be in one place and you just pass in the proper motor